src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -459,7 +459,8 @@ private string UFormatDateString(DateTime dateTime)
                             break;
 
                         case 'V':
-                            sb.Append((dateTime.DayOfYear / 7) + 1);
+                            GregorianCalendar calendar = new GregorianCalendar();
```

Comment:
I believe we shouldn't create new object.
`System.Threading.Thread.CurrentThread.CurrentCulture.Calendar`